### PR TITLE
Moved all the template creation tasks from thoth-ops

### PR DESCRIPTION
**Pull request by harshad16**
Moved all the template creation tasks from thoth-ops
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -47,6 +48,26 @@
...
+    - https://raw.githubusercontent.com/thoth-station/adviser/master/openshift/job-provenanceChecker-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/adviser/master/openshift/job-adviser-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/adviser/master/openshift/job-dependencyMonkey-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/amun-api/master/openshift/inspectJob-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/amun-api/master/openshift/inspectJobWithCPU-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/package-extract/master/openshift/job-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/package-analyzer/master/openshift/job-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/graph-sync-job/master/openshift/adviserSyncJob-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/graph-sync-job/master/openshift/provenanceCheckerSyncJob-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/graph-sync-job/master/openshift/solverSyncJob-template.yaml
```
Comment (bissenbay) [inline tasks/main.yml]: @harshad16 It looks like we have it on line #49?

File: tasks/main.yml [hunk 2]
```diff
@@ -47,6 +48,26 @@
...
+    - https://raw.githubusercontent.com/thoth-station/graph-sync-job/master/openshift/packageExtractSyncJob-template.yaml
```
Comment (bissenbay) [inline tasks/main.yml]: @harshad16 this as well

File: tasks/main.yml (hunk 1 above)
Comment (harshad16) [reply tasks/main.yml]: @bissenbay  hey, thanks for pointing that out :smile: . Fixed it

Comment (bissenbay) [reply tasks/main.yml]: thanks

**Review by bissenbay — APPROVED** [review]
:100:

**Pull request merged**